### DISCUSSION TITLE
Set aspect ratio to "equal" by default.

### DISCUSSION THREAD

**Discussion by tylerflex** [pydata/xarray#6168] [Q&A]
I'm building an application around xarray and want the `xr.DataArray.plot.pcolormesh()` outputs to have equal aspect ratio (same scaling of x units to y units) by default.  

One way is to grab the axes object (`ax`) and apply `ax.set_aspec('equal')` after plotting, but I'm wondering whether there is any more convenient way to instead set this as a global parameter in xarray.

If all else fails, perhaps I can subclass the DataArray but I'd like to not have to [per this discussion](https://xarray.pydata.org/en/stable/internals/extending-xarray.html).

Thank you!

**Comment by dcherian** [comment]
>  I'm wondering whether there is any more convenient way to instead set this as a global parameter in xarray.

There is none. I would explore a matplotlib based solution.
